install/0001_00_machine-config-operator_00_service.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     k8s-app: machine-config-daemon
   annotations:
     service.beta.openshift.io/serving-cert-secret-name: proxy-tls
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
```

Comment:
And similar for this.

Reply:
@cgwalters do you mean that the mcd itself should be available?

Reply:
Right, the MCD isn't control plane. so we don't want to exclude anything related to it.

Reply:
Thanks, updated